internal/pkg/template/input/rules.go

```diff
@@ -24,11 +25,20 @@ func (r Rules) Empty() bool {
 	return len(r) == 0
 }
 
-func (r Rules) ValidateValue(userInput interface{}, fieldId string) (err error) {
+func (r Rules) ValidateValue(input Input, value any) (err error) {
+	// Skip empty rules
 	if r.Empty() {
 		return nil
 	}
 
+	// Convert rules to string
+	rules := string(r)
+
+	// Handle required object: empty object "{}" is valid for "required" rule, so we have to add custom rule.
+	if input.Type == TypeObject {
+		rules = "requiredNotEmpty," + rules
+	}
+
```

Comment:
Ak mame input `type=object`, a nastavime `rules=required`, a hodnota je  prazdny JSON objekt `{}` (prazdna map), tak to neskonci chybou.
Kedze `goplayground/validator` to neberie ako prazdu hodnotu

Tu som to fixol, vytvoril som nove pravidlo, ktore sa automaticky doplní k povodnym.